### PR TITLE
Update all dependencies to latest version

### PR DESCRIPTION
**Pull request by cory-tanner**

### CODE REVIEW

**Comment by cory-tanner** [comment]
I tested these changes locally and all plugins look to be working. Would love a second pair of eyes :)

File: package.json [hunk 1]
```diff
@@ -14,15 +14,15 @@
...
     "object-assign": "^4.1.1",
-    "postcss": "^6.0.14",
-    "postcss-custom-properties": "^6.2.0",
-    "postcss-discard-comments": "^2.0.4",
-    "postcss-hexrgba": "^1.0.0",
-    "postcss-inline-svg": "^3.0.0",
-    "postcss-nested": "^2.1.2",
+    "postcss": "^7.0.18",
+    "postcss-custom-properties": "^8.0.11",
+    "postcss-discard-comments": "^4.0.2",
```
Comment (monicawheeler) [inline package.json:21]: The `postcss-discard-comments` package is listed as deprecated. Should we be using something else here? In any event, it works properly 👍 

https://github.com/ben-eb/postcss-discard-comments

File: package.json [hunk 2]
```diff
@@ -14,15 +14,15 @@
...
+    "postcss-hexrgba": "^1.0.2",
+    "postcss-inline-svg": "^3.1.1",
+    "postcss-nested": "^4.1.2",
     "postcss-partial-import": "^4.1.0",
-    "postcss-svgo": "^2.1.6"
+    "postcss-svgo": "^4.0.2"
```
Comment (monicawheeler) [inline package.json:26]: The `postcss-svgo` package is also listed as deprecated. Should we be using the referred package in this repo? Also, this one works as intended as well 👍 

https://github.com/ben-eb/postcss-svgo

File: package.json (hunk 1 above)
Comment (monicawheeler) [reply package.json:21]: Leaving as is. Closing comment.

File: package.json (hunk 2 above)
Comment (monicawheeler) [reply package.json:26]: Leaving as is. Closing comment.

**Review by monicawheeler — APPROVED** [review]
I tested every update locally within my project app and all is working as expected. ✅

**Pull request merged**